components/raftstore-v2/src/worker/tablet.rs

```diff
@@ -390,12 +391,16 @@ impl<EK: KvEngine> Runner<EK> {
     /// Returns true if task is consumed. Failure is considered consumed.
     fn process_destroy_task(logger: &Logger, registry: &TabletRegistry<EK>, path: &Path) -> bool {
         match EK::locked(path.to_str().unwrap()) {
-            Err(e) => warn!(
-                logger,
-                "failed to check whether the tablet path is locked";
-                "err" => ?e,
-                "path" => path.display(),
-            ),
+            Err(e) if !e.to_string().contains("No such file or directory") => {
```

Comment:
Is there other better way to check if it's "no such file" error as I think error message may be changed over time?

Reply:
It's RocksDB error message, no other way.